### PR TITLE
Fixing the output of code examples in the preprocessing chapter

### LINKED ISSUE #17161: Inconsistent output of code example in Proprocessing Chapter in Docu
The following [docu](https://huggingface.co/docs/transformers/preprocessing) contains some inconsistent/wrong output examples for the `Build Tensor` subsection:

```python
batch_sentences = [
    "But what about second breakfast?",
    "Don't think he knows about second breakfast, Pip.",
    "What about elevensies?",
]
encoded_input = tokenizer(batch, padding=True, truncation=True, return_tensors="pt")
print(encoded_input)
```
Output:
```txt
{'input_ids': tensor([[  101,   153,  7719, 21490,  1122,  1114,  9582,  1623,   102],
                      [  101,  5226,  1122,  9649,  1199,  2610,  1236,   102,     0]]), 
 'token_type_ids': tensor([[0, 0, 0, 0, 0, 0, 0, 0, 0],
                           [0, 0, 0, 0, 0, 0, 0, 0, 0]]), 
 'attention_mask': tensor([[1, 1, 1, 1, 1, 1, 1, 1, 1],
                           [1, 1, 1, 1, 1, 1, 1, 1, 0]])}
```

The output does not contain the applied truncation and padding and also misses the last sentence of `batch_sentences`. This also applies to the TensorFlow example.

### PR DESCRIPTION
**Pull request by HallerPatrick**
# What does this PR do?

Fixes #17161 (issue)

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [x] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/main/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [x] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/main/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/main/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

@sgugger

### CODE REVIEW

**Review by sgugger — APPROVED** [review]
Thanks a lot for your PR!
LGTM, but but let's wait for @stevhliu to have a look before merging.

**Review by stevhliu — APPROVED** [review]
Thanks for catching this error, looks good to me :)
